src/Core/Addon/Module/ModuleManagerBuilder.php

```diff
@@ -159,13 +166,18 @@ private function getSymfonyRouter()
     {
         // get the environment to load the good routing file
         $routeFileName = _PS_MODE_DEV_ === true ? 'routing_dev.yml' : 'routing.yml';
-        $routesDirectory = _PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'app'.DIRECTORY_SEPARATOR.'config';
+        $routesDirectory = $this->getConfigDir();
         $locator = new FileLocator(array($routesDirectory));
         $loader = new YamlFileLoader($locator);
 
         return new Router($loader, $routeFileName);
     }
 
+    protected function getConfigDir()
```

Comment:
It's a single reuse of strings concatenation. There is no need to have a more generic class (PrestaShop\PrestaShop\Core\Addon\Module\ModuleManager doesn't currently extend any other class).